frontend/src/components/Errors/index.js

```diff
@@ -0,0 +1,9 @@
+import ErrorGeneral from './ErrorGeneral.js';
+import ErrorNotCollaborator from './ErrorNotCollaborator.js';
+import ErrorTripNotFound from './ErrorTripNotFound.js';
+
+export {
+  ErrorGeneral,
+  ErrorNotCollaborator,
+  ErrorTripNotFound
+};
```

Comment:
what does this file do? why is it needed?

Reply:
Each error page is put into its own separate file, so the index file just grabs all of them and exports them for external use.